### PR TITLE
Torrent details

### PR DESCRIPTION
**Pull request by dcposch**
Show file list, open folder containing downloaded files, open individual files, play/pause individual videos.

image

image

### CODE REVIEW

**Comment by ChrisMorrisOrg** [comment]
This looks so much neater!

File: renderer/views/torrent-list.js [hunk 1]
```diff
@@ -5,165 +5,205 @@ var hyperx = require('hyperx')
...
+    }
+    return hx`
+      <tr onclick=${handleClick}>
+        <td class='col-icon'>
+          <i class='icon ${iconClass}'>${icon}</i>
+        </td>
+        <td class='col-name'>${file.name}</td>
+        <td class='col-progress'>${progress}</td>
+        <td class='col-size'>${prettyBytes(file.length)}</td>
+      </li>
```
Comment (ChrisMorrisOrg) [inline renderer/views/torrent-list.js]: Is `</li>` meant to be here?

Comment (dcposch) [inline renderer/views/torrent-list.js]: uhhh no good call

**Comment by feross** [comment]
:+1: :+1: :+1:

**Pull request merged**